nixos/modules/services/security/torify.nix

```
@@ -25,6 +25,7 @@ in
     services.tor.tsocks = {
 
       enable = mkOption {
+        type = lib.types.bool;
```

```suggestion
        type = types.bool;
```
There is a `with lib` at the top of the module.